src/base/bittorrent/sessionimpl.cpp

```diff
@@ -2680,11 +2680,14 @@ bool SessionImpl::addTorrent_impl(const std::variant<MagnetUri, TorrentInfo> &so
     if (m_loadingTorrents.contains(id) || (infoHash.isHybrid() && m_loadingTorrents.contains(altID)))
         return false;
 
-    if (Torrent *torrent = findTorrent(infoHash); torrent)
+    if (Torrent *torrent = findTorrent(infoHash))
     {
         // a duplicate torrent is being added
         if (torrent->isPrivate())
+        {
+            LogMsg(tr("Torrent '%1' is already in the transfer list. Trackers cannot be merged because it is a private torrent.").arg(torrent->name()));
```

Comment:
```suggestion
            LogMsg(tr("Torrent is already in the transfer list. Trackers cannot be merged because it is a private torrent. Torrent: \"%1\"").arg(torrent->name()));
```
It keeps the message readable when the torrent name is long and messy.